src/pages/CustomerInvoicePage.js

```diff
@@ -12,6 +12,7 @@ import { SearchBar } from 'react-native-ui-components';
 
 import { MODAL_KEYS, getModalTitle } from '../utilities';
 import { buttonStrings, modalStrings } from '../localization';
+import { DEFAULT_KEY_EXTRACTOR, DEFAULT_GET_ITEM_LAYOUT } from './dataTableUtilities/utilities';
```

Comment:
Looks weird to me to name function variable exports in UPPER_SNAKE_CASE, I'd normally reserve that for sources of string constants. What do you reckon?

Reply:
yup I agree - any suggestion though? Obviously landed here cause I was out of ideas. I'll update SI if/when I commit new names 🤣

Reply:
Just regular `keyExtractor` or maybe `recordKeyExtractor`/`idExtractor`.

Basically camelCase is fine?